<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtMonitor</class>
 <widget class="QMainWindow" name="qtMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Band</string>
    </property>
    <widget class="QPushButton" name="band_160_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>160</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_80_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>80</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_60_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_40_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_30_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_20_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_17_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>17</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_15_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_12_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_10_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_6_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="band_gen_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Gen</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QPushButton" name="mode_lsb_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>LSB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_usb_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_dsb_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>DSB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_cwl_pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>CWL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_cwu_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>CWU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_am_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>AM</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="vfo_lcdNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>10</number>
    </property>
    <property name="value" stdset="0">
     <double>7.056000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="serverLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Server:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rx_spinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="spectrum" name="spectrumFrame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="waterfall" name="waterfallFrame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="audioComboBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>spectrum</class>
   <extends>QFrame</extends>
   <header>spectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>waterfall</class>
   <extends>QFrame</extends>
   <header>waterfall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
